src/libos/Makefile

```diff
@@ -106,13 +106,19 @@ $(LIBOS_CORE_A): $(LIBOS_CORE_RS_A) $(C_OBJS) $(CXX_OBJS) $(S_OBJS) $(EDL_C_OBJS
 	@ar r $@ $(C_OBJS) $(CXX_OBJS) $(S_OBJS) $(EDL_C_OBJS)
 	@echo "GEN => $@"
 
+RUSTC_WRAPPER :=
+
+ifneq (, $(shell which sccache))
+	RUSTC_WRAPPER=sccache
```

Comment:
There is a difference between `var := val` and `var = val` in Makefile. The style of this Makefile is to prefer the former one.